drivers/soc/qcom/peripheral-loader.c

```diff
@@ -1678,8 +1677,10 @@ static int __init msm_pil_init(void)
 	for (i = 0; i < resource_size(&res)/sizeof(u32); i++)
 		writel_relaxed(0, pil_info_base + (i * sizeof(u32)));
 
-	/* Get Global minidump ToC*/
 #ifndef TARGET_NO_MINIDUMP_SUPPORT
+	/* Get Global minidump ToC*/
+	size_t size;
+
```

Comment:
C99 disallows mixed declarations and code, defining `TARGET_NO_MINIDUMP_SUPPORT` should throw a build error here. Iirc kernel guys commonly use:
```c
#else
	(void)size;
#endif
```
at the bottom, but don't quote me on that.

Reply:
I will choose your second suggestion.

Part of my build-test was to check the build log for occurrences of "[-W". Don't know why it didn't show up.

Reply:
It doesn't show up because `TARGET_NO_MINIDUMP_SUPPORT` is defined by default for all SODP devices, at the top of the file ;)

I have manually undefined it to show you the effects, and remind myself that it's C90, not C99 :wink

Reply:
Ah thought it would be then a warning from the preprocessor. Tbh I don't know much about C... But now I learned something :-)